packages/bundler-plugin-core/src/types.ts

```diff
@@ -278,6 +278,21 @@ export interface Options {
     excludeReplayWorker?: boolean;
   };
 
+  /**
+   * Options related to component name annotations.
+   * Disabled by default, unless a value is set for this option.
+   * Please note that this feature is not currently supported by the esbuild bundler plugins, and will only annotate React components
+   */
+  componentNameAnnotate?: {
```

Comment:
The name of this option should properly reflect that it will only annotate components in react applications. Probably something like `reactComponentAnnotation`.

I would also hoist the JS doc for the enabled option to this level and only explain what enabled does in the enabled js doc.

Reply:
I kept the name generic for now since we eventually may extend it to work for other frameworks / libraries, and didn't want to have users be forced to change the option in their config when that does happen.

Noted on the docs, will change that!

Reply:
alright I'll change it 👍